BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -22,7 +22,7 @@ class COMPONENT_ENTRY (Structure):
         ('size',        c_uint32),   # Region/Component size in byte
         ('attribute',   c_uint8),    # Attribute:  BIT7 Reserved component entry
         ('alignment',   c_uint8),    # This image need to be loaded to memory  in (1 << Alignment) address
-        ('auth_type',   c_uint8),    # Refer AUTH_TYPE_VALUE: 0 - "NONE"; 1- "SHA2_256";  2- "RSA2048SHA256"; 3- "SHA2_384"; 4 - RSA3072SHA384
+        ('auth_type',   c_uint8),    # Refer AUTH_TYPE_VALUE: 0 - "NONE"; 1- "SHA2_256";  2- "SHA2_384";  3- "RSA2048SHA256"; 4 - RSA3072SHA384

```

Comment:
Better add available strings in the help, else it is difficult for user to know these strings.

Reply:
Yes, useing "choice" will resolve this issue.

Reply:
will add choices.